modules/nf-core/modules/bwamem2/mem/main.nf

```diff
@@ -10,31 +10,33 @@ process BWAMEM2_MEM {
     input:
     tuple val(meta), path(reads)
     path  index
+    val   sort_bam
 
     output:
     tuple val(meta), path("*.bam"), emit: bam
     path  "versions.yml"          , emit: versions
 
     script:
-    def split_cpus = Math.floor(task.cpus/2)
-    def args       = task.ext.args  ?: ''
-    def args2      = task.ext.args2 ?: ''
-    def prefix     = task.ext.prefix ?: "${meta.id}"
+    def args = task.ext.args ?: ''
+    def args2 = task.ext.args2 ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
     def read_group = meta.read_group ? "-R ${meta.read_group}" : ""
```

Comment:
should this go to the `modules.config` as well? seem to be the same to me as the `status` line basically

Reply:
The read_group you mean?